File: apps/website/mfe/src/app/pages/instant-quote/models/space.model.ts
import { IResponseModel } from './response.model';

export interface ISpaceModel {
  id: string;
  name: string;
  maxHours: number;
  minHours: number;
  isSelected?: boolean;
}

export interface SpaceResponse extends IResponseModel {
  data: Array<SpaceModel>;
}

export interface SpaceModel {
  id: string;
  name: string;
  maxHours: number;
  minHours: number;
}

File: apps/api/Models/SpaceModel.cs
using Google.Cloud.Firestore;

namespace Space.Model;

[FirestoreData]
public class SpaceModel
{
    [FirestoreProperty("id")]
    public string? ID { get; set; }

    [FirestoreProperty("name")]
    public string? Name { get; set; }

    [FirestoreProperty("maxHours")]
    public long MaxHours { get; set; }

    [FirestoreProperty("minHours")]
    public long MinHours { get; set; }
}